### ISSUE TITLE
Full results for GEMMA mapping in GN2

### ISSUE THREAD

**Issue by Dashbrook** [genenetwork/genenetwork2#266]
Would it be possible to add an option to download the full results from mapping in GN2, not just the top 200 as currently shown? This is needed for example when wanting to do a direct comparison with a marker found with GN1 mapping. It is a minor issue, but hopefully easy to solve. 

Thank you,
David

**Comment by zsloan** [comment]
Thanks for reminding me of this, as this issue actually came up in the
past. I'll add an option to download results either today or tomorrow.

**Comment by Dashbrook** [comment]
That's great, thank you!

**Comment by zsloan** [comment]
I added an option to download mapping results as CSV (it's in the top left
below the trait info). If it automatically opens in Excel you might need to
select the column, click "data", and tell it to be tab delimited. I could
also switch to comma delimited if that's better; I kind of arbitrarily
chose tabs, but it's an easy change to make.

Let me know if you encounter any errors or if the output isn't what you
expect. Since I just added this today and there are a number of mapping
variations, it's possible I missed something.
